### PR TITLE
Update Azure.Storage.Blobs.Batch.csproj

### PR DESCRIPTION
**Pull request by AlexGhiondea**
Remove the duplicate links.

/cc @weshaggard @tg-msft @ShivangiReja @JoshLove-msft

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj [hunk 1]
```diff
@@ -14,8 +14,6 @@
       This client library allows you to batch multiple Azure Blob Storage operations in a single request.
       For this release see notes - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batch/README.md and https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md
       in addition to the breaking changes https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batch/BreakingChanges.txt
-      For this release see notes - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batching/README.md and https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batching/Changelog.txt
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj]: Worth yanking the preview3 link above while we're doing this?

**Review by tg-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj (hunk 1 above)
Comment (AlexGhiondea) [reply sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj]: What would be the replacement?

Comment (tg-msft) [reply sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj]: I'd just kill the whole section unless we have a new aka.ms link?

**Review by amnguye — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj (hunk 1 above)
Comment (AlexGhiondea) [reply sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj]: Done.

**Pull request merged**